debian/control

```diff
@@ -266,19 +266,19 @@ Description: Display server for Ubuntu - shared library
  Contains the shared libraries required for the Mir server and client.
 
 # Longer-term these drivers should move out-of-tree
-Package: mir-platform-graphics-mesa-x16
+Package: mir-platform-graphics-x16
```

Comment:
We probably want to Break and/or Replace mir-platform-graphics-mesa-x16

(Although should we really still be on 16? There ought to be incompatibilities with 1.8)

Reply:
There's no good reason to automatically remove `mir-platform-graphics-mesa-x16` - it will become eligible for autoremoval.

It should, indeed, be x17. The BufferAllocator PR bumped mirplatform ABI, but must have missed this.

Reply:
It was less about autoremoval than about probing both platforms if they are both installed. (Which can happen.) But if we bump the ABI we can only pick the right one.